ironfish/src/wallet/account.ts

```diff
@@ -112,7 +106,34 @@ export class Account {
           unconfirmedBalance += new Note(decryptedNote.serializedNote).value()
         }
 
-        this.saveDecryptedNoteSequence(decryptedNote.transactionHash, hash)
+        const nullifierHash = decryptedNote.nullifierHash
```

Comment:
this is opinionated nit but can we early return on the node id and just unindent this whole section?